utils/gyb.py

```diff
@@ -716,7 +716,8 @@ def execute(self, context):
 
         # If we got a result, the code was an expression, so append
         # its value
-        if result is not None and result != '':
+        if result is not None \
+                or (isinstance(result, basestring) and result != ''):
```

Comment:
Done. I'm surprised too, pleasantly. The `util/gyb` file explicitly referencing python2.7 and discussion in #1535 threw me off. I guess I also expected to see `unicode_literals` imported if the code was trying to be Python 3 ready, and didn't really pay attention to the rest of the code.